bolt-simple-taproot.md

```diff
@@ -846,19 +855,21 @@ Accepted HTLCs inherit a similar format:
     * `htlc_script_root = tapscript_root([htlc_timeout, htlc_success])`
     * `htlc_timeout`:
         ```
-        <remote_htlcpubkey>
-        OP_CHECKSIGVERIFY
-        1 OP_CHECKSEQUENCEVERIFY OP_DROP
-        <cltv_expiry>
-        OP_CHECKLOCKTIMEVERIFY
+        <remote_htlcpubkey> OP_CHECKSIG
+        OP_CHECKSEQUENCEVERIFY
+        <cltv_expiry> OP_CHECKLOCKTIMEVERIFY OP_DROP
         ```
     * `htlc_success`:
         ```
         OP_SIZE 32 OP_EQUALVERIFY OP_HASH160 <RIPEMD160(payment_hash)> OP_EQUALVERIFY
-        <local_htlcpubkey> OP_CHECKSIGADD <remote_htlcpubkey> OP_CHECKSIGADD 2 OP_EQUALVERIFY
-        1 OP_CHECKSEQUENCEVERIFY
+        <local_htlcpubkey> OP_CHECKSIGVERIFY
+        <remote_htlcpubkey> OP_CHECKSIG
         ```
 
+Note that there is no `OP_CHECKSEQUENCEVERIFY` in the offered HTLC's timeout path
```

Comment:
Thanks for this note, makes sense, and isn't too big of a departure w.r.t how things work today.